components/logins/src/store.rs

```diff
@@ -120,6 +120,19 @@ impl LoginStore {
         self.db.lock().unwrap().update(login)
     }
 
+    // TODO: STUB FOR THE NEW API THAT WILL BE USED AND WILL BE CHANGED
+    pub fn add_or_update(&self, login: Login) -> Result<()> {
```

Comment:
We decided to keep the `add` and `update` functions separate in order to avoid the necessary UI updates so I think you don't need this stub.